### PR TITLE
Climbing Route Models

### PR DESCRIPTION
**Pull request by wulalasuperdope**
- fetch this pr
- run the app
- go to each climbing route model and you should be able to see seeded 6 models

P.S. Route model created through DnD might not display, since I added 2 more hold types.

### CODE REVIEW

**Review by nries1 — COMMENTED** [review]
OMG THIS IS AMAZING!!

**Review by nries1 — COMMENTED** [review]
2 Questions on the route model table:
1. Does Route Model need to have the grade and the hold color? Aren't we getting that from the climbing route table?
2. Can the holdModelType column be stored in the hold table? It would  make our database more normalized because we wouldn't be listing the holdModelType more than once for each hold

**Comment by wulalasuperdope** [comment]
> 2 Questions on the route model table:
> 
> 1. Does Route Model need to have the grade and the hold color? Aren't we getting that from the climbing route table?

**I think Route Model needs to have grade and color. Since essentially the table is about hold models and the way to connect it to Climbing Route table is grade and color.**

> 2. Can the holdModelType column be stored in the hold table? It would  make our database more normalized because we wouldn't be listing the holdModelType more than once for each hold

**The holdModelType column is stored in the Hold table as "ModelType". Since I used 2 model types for foot hold and 2 for sloper, I need to use holdType and holdModelType to connect the Hold table. But I can simplify this connection.**

File: seed-data.js [hunk 1]
```diff
@@ -30,15 +30,11 @@ const holds = [
     modelType: 'a-octahedron'
   },
   {
-    holdType: 'footHold',
-    modelType: 'a-sphere'
-  },
-  {
-    holdType: 'sloper',
+    holdType: 'sloper (sphere)',
```
Comment (nries1) [inline seed-data.js:33]: is the route canvas ready to accept this new naming convention?

**Review by nries1 — APPROVED** [review]

**Pull request merged**